.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,58 @@
+name: Feature Request
+description: Request a new feature
+labels: ["Type: enhancement"]
```

Comment:
It would be nice if we didn't mix vocabulary here. "Enhancement" is broader than "new feature" (an enhancement can be a performance improvement, for example).